src/core_plugins/timelion/public/chain.peg

```diff
@@ -21,7 +21,7 @@
 }
 
 start
- = tree:series+ {
```

Comment:
any reason why this is removed? Don't entirely see the relation with the other changes.